wasmbinding/message_plugin.go

```diff
@@ -271,11 +271,11 @@ func (m *CustomMessenger) submitProposal(ctx sdk.Context, contractAddr sdk.AccAd
 	return nil, [][]byte{data}, nil
 }
 
-func (m *CustomMessenger) PerformSubmitProposal(ctx sdk.Context, contractAddr sdk.AccAddress, submitProposal *bindings.SubmitProposal) (*admintypes.MsgSubmitProposalResponse, error) {
+func (m *CustomMessenger) PerformSubmitAdminProposal(ctx sdk.Context, contractAddr sdk.AccAddress, submitAdminProposal *bindings.SubmitAdminProposal) (*admintypes.MsgSubmitProposalResponse, error) {
```

Comment:
please make the method private

Reply:
would be grateful if you could make the rest of the `Perform...` methods private as well since they should be private